### PR TITLE
fingertip: print information when executed without args

### PR DESCRIPTION
**Pull request by shigaraki0**
* Removed log.error message
* Print examples when fingertip is executed with no args

### CODE REVIEW

**Comment by t184256** [comment]
1) Why the empty lines before and after?
2) Please use a multiline string:

```
USAGE = '''
Name: \n\tfingertip ...
'''

...

lalala:
    print(USAGE)
```

**Comment by shigaraki0** [comment]
@t184256 : https://github.com/t184256/fingertip/pull/28/commits/021a27df65d19ca38bd53d04633e26cb5d94a9bb

**Comment by t184256** [comment]
move it outside the function or use textwrap.dedent, otherwise it'll have tons of spaces inside

File: fingertip/main.py [hunk 1]
```diff
@@ -46,7 +47,17 @@ def main():
     args = sys.argv[1:]
 
     if not args:
-        fingertip.util.log.error('no plugin specified')
+        info = """ Name:
+        fingertip
+ Version:"""
+        version = subprocess.check_output(['rpm', '-q', 'fingertip']).decode("utf-8").replace("fingertip-", "")
```
Comment (t184256) [inline fingertip/main.py:53]: nope, nope, nope, why do you think the system has `rpm`?

Comment (t184256) [reply fingertip/main.py:53]: In short, there's no good way to know the version.

**Comment by shigaraki0** [comment]
@t184256: https://github.com/t184256/fingertip/pull/28/commits/f32ac75354fc5cb8d5f94d85b9f9ea1a11af25fc

File: fingertip/main.py [hunk 2]
```diff
@@ -46,7 +47,15 @@ def main():
     args = sys.argv[1:]
 
     if not args:
-        fingertip.util.log.error('no plugin specified')
+        info = """ Name:
+        fingertip
+ Usage:
```
Comment (t184256) [inline fingertip/main.py:52]: 1) Why is Usage not indented?
2) could be "fingertip: a tool to control VMs/containers"
3) usage messages sometimes use `sys.argv[0]` as a command to match how the user has invoked it

Also, would be cool to stay under 79 character line limit, but I'm not sure this is reasonable in this particular case.

**Comment by shigaraki0** [comment]
@t184256

File: fingertip/main.py [hunk 3]
```diff
@@ -46,7 +47,16 @@ def main():
...
+        info = """
+        Name:
+                fingertip - a tool to control VMs/containers
+        Usage:
+                fingertip os.fedora + script.debug myscript.sh # checkpoint-powered debugger
+                fingertip os.fedora --version=34 + script.debug myscript.sh # install Fedora 34 and debug a script
+                fingertip os.fedora + ssh  # install Fedora and SSH into it
+                fingertip cleanup machines all # empty your ~/.cache/fingertip/machines often, at least after each update
+                fingertip cleanup everything # clean ~/.cache/fingertip occasionally"""
+        print(textwrap.dedent(info))
```
Comment (t184256) [inline fingertip/main.py:59]: `print(textwrap.dedent(info).lstrip())` to get rid of the empty line in output and it'll be fine

**Comment by shigaraki0** [comment]
@t184256 fixed

**Pull request merged**